Add edit button linking to github and format some files

awennersteen opened the pull request:
Three possibilities, you have write access to repo (currently the two of us), you have a github account but no write access to the repo, and you don't have a github account/isn't logged in.

1. Write access
    1. You'll be able to edit the file directly and save in the repo on github
1. No write access
    1. Github explains to you how to fork the repo and let's you fork and edit
1. No Github or not logged in to github
   1. you're met with the signup or login screen
   

It looks like this, IDK if it will look better perhaps to right-align the button? Maybe make it stand out more?
   
![image](https://github.com/tramitit/tramitit.github.io/assets/26585675/75140987-ecbf-4d77-a508-af962db375d9)

Commit:
Add edit button linking to github and format some files

rbatista191 commented:
@awennersteen looks great!
I would probably add another block at the end (between the markdown and Disqus) with a `Improve this article by editing it on GitHub`:

![Screenshot 2024-06-13 at 15 15 05](https://github.com/tramitit/tramitit.github.io/assets/138892976/341b8c70-f486-435c-8de4-6c8524113fab)

I will try to create the guide today, which can also be added to these individual pages.

Commit:
Move button to right, add extra call-to-contribute

Commit:
Support i18n

awennersteen commented:
Alright, in the end I decided that it made more sense to put the edit button on the right (just seems more conventional), and I added the call to action at the bottom. 
I also fixed some other i18n things that I realized that we hadn't handled. We currently only do this for en and fr so should generate for the rest too one by one.

![image](https://github.com/tramitit/tramitit.github.io/assets/26585675/d6101fa5-1a3c-46d1-9b26-ea6a98f0f4ee)
![image](https://github.com/tramitit/tramitit.github.io/assets/26585675/325997c8-1025-47dd-8a8a-2b0e44777e39)

awennersteen commented on themes/hugoplate/layouts/partials/essentials/head.html:
```diff
@@ -29,7 +31,7 @@
 
 <!-- google analytics -->
 {{ if and site.Config.Services.GoogleAnalytics.ID (ne site.Config.Services.GoogleAnalytics.ID "G-G6825VQMZB") }}
-<!-- {{ template "_internal/google_analytics.html" . }} -->
+  <!-- {{ template "_internal/google_analytics.html" . }} -->
 {{ end }}
 
```
we don't use this one right, since we use gtm
I think I'll go and remove things from the themes from now on whenever I stumble over them

rbatista191 replied:
Yes, feel free to do so.

rbatista191 commented:
> Alright, in the end I decided that it made more sense to put the edit button on the right (just seems more conventional), and I added the call to action at the bottom. 

Like it a lot more!

> I also fixed some other i18n things that I realized that we hadn't handled. We currently only do this for en and fr so should generate for the rest too one by one.

Not sure I follow what you mean by `We currently only do this for en and fr` but we can discuss tomorrow.
